e2e/policy/policy_test.go

```diff
@@ -145,6 +145,21 @@ func TestPolicy(t *testing.T) {
 		// errors should happen
 		require.Greater(newFailures, initialFailures)
 	})
+
+	t.Run("cli does not verify coordinator with unexpected policy hash", func(t *testing.T) {
+		require := require.New(t)
+
+		// change expected coordinator policy hash
+		policyHash, err := os.ReadFile(path.Join(ct.WorkDir, "coordinator-policy.sha256"))
+		require.NoError(err)
+		require.NoError(os.WriteFile(path.Join(ct.WorkDir, "coordinator-policy.sha256"), []byte("0000000000000000000000000000000000000000000000000000000000000000"), 0o644))
```

Comment:
I would prefer a more plausible hash - all-zeroes could ostensibly have some special meaning. Could we maybe just flip a bit in the expected one?